bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -459,10 +459,20 @@ object TimeProblems {
       }
 
   val SCHEDULING_SECTION = "Time Requests"
-}
 
-import ProblemRobot._
-import TimeProblems._
+  // REL-2032 Check that none of the requested times per partner are zero
+  def partnerZeroTimeRequest(p: Proposal, s:ShellAdvisor): List[ProblemRobot.Problem] = p.proposalClass match {
+    case g:GeminiNormalProposalClass =>
+      val probs = for {
+          sub <- g.subs.swap.right
+        } yield for {
```

Comment:
The first one gets you out of the `Either`, the second iterates over sub which is a `List[NgoSubmission]`